packages/kbn-esql-validation-autocomplete/src/definitions/aggs.ts

```diff
@@ -16,7 +16,7 @@ function createNumericAggDefinition({
 }: {
   name: string;
   description: string;
-  args?: Array<{ name: string; type: string; value: string; literalOnly?: boolean }>;
+  args?: Array<{ name: string; type: string; value: string; constantOnly?: boolean }>;
```

Comment:
Actually, I was hoping this change would align us closer with ES since they use "constant" in their error messages. Wasn't aware of the Monaco thing.

```
[esql] > Unexpected error from Elasticsearch: verification_exception - Found 1 problem
line 1:51: second argument of [bucket(bytes, bytes, 30, 20)] must be a constant, received [bytes]
```


WDYT?

Reply:
Yes makes sense 👍